apps/dav/tests/unit/CalDAV/Schedule/IMipPluginTest.php

```diff
@@ -890,4 +890,23 @@ public function testNoButtons(): void {
 		$this->plugin->schedule($message);
 		$this->assertEquals('1.1', $message->getScheduleStatus());
 	}
+
+	public function testImipDisabledForEvent(): void {
+		// construct iTip message with event and attendees
+		$calendar = new VCalendar();
+		$calendar->add('VEVENT', ['UID' => 'uid-1234']);
+		$event = $calendar->VEVENT;
+		$event->add('ORGANIZER', 'mailto:gandalf@wiz.ard');
+		$event->add('ATTENDEE', 'mailto:' . 'frodo@hobb.it', ['RSVP' => 'TRUE',  'CN' => 'Frodo']);
+		$event->add('X-NC-DISABLE-SCHEDULING', '1');
+		$message = new Message();
+		$message->method = 'REQUEST';
+		$message->message = $calendar;
+		$message->sender = 'mailto:gandalf@wiz.ard';
+		$message->senderName = 'Mr. Wizard';
+		$message->recipient = 'mailto:' . 'frodo@hobb.it';
+		
+		$this->plugin->schedule($message);
+		$this->assertEquals('1.0', $message->getScheduleStatus());
```

Comment:
```suggestion
		$this->assertEquals('1.0;We got the message, but iMip message are disabled for this event', $message->scheduleStatus);
```

Just to make the test a bit more robust.